docs/snippets/tracer/manual.ts

```diff
@@ -3,30 +3,30 @@ import { Tracer } from '@aws-lambda-powertools/tracer';
 const tracer = new Tracer({ serviceName: 'serverlessAirline' });
 
 export const handler = async (_event: any, context: any): Promise<unknown> => {
-    const segment = tracer.getSegment(); // This is the facade segment (the one that is created by AWS Lambda)
-    // Create subsegment for the function & set it as active
-    const subsegment = segment.addNewSubsegment(`## ${process.env._HANDLER}`);
-    tracer.setSegment(subsegment);
+  const segment = tracer.getSegment(); // This is the facade segment (the one that is created by AWS Lambda)
+  // Create subsegment for the function & set it as active
+  const subsegment = segment.addNewSubsegment(`## ${process.env._HANDLER}`);
+  tracer.setSegment(subsegment);
 
-    // Annotate the subsegment with the cold start & serviceName
-    tracer.annotateColdStart();
-    tracer.addServiceNameAnnotation();
+  // Annotate the subsegment with the cold start & serviceName
+  tracer.annotateColdStart();
+  tracer.addServiceNameAnnotation();
 
-    let res;
-    try {
-        /* ... */
-        // Add the response as metadata 
-        tracer.addResponseAsMetadata(res, process.env._HANDLER);
-    } catch (err) {
-        // Add the error as metadata
-        tracer.addErrorAsMetadata(err as Error);
-        throw err;
-    } finally {
-        // Close subsegment (the AWS Lambda one is closed automatically)
-        subsegment.close();
-        // Set back the facade segment as active again
-        tracer.setSegment(segment);
-    }
+  let res;
+  try {
+    /* ... */
+    // Add the response as metadata 
+    tracer.addResponseAsMetadata(res, process.env._HANDLER);
```

Comment:
Pass directly an object here and remove the `res` constant from example, as well as the empty comment below